tools/publish_dcos_file.py

```diff
@@ -50,32 +51,33 @@ def __init__(
                 )
             )
 
-        s3_bucket = os.environ.get('S3_BUCKET')
-        if not s3_bucket:
-            s3_bucket = 'infinity-artifacts'
+        s3_bucket = os.environ.get('S3_BUCKET', 'infinity-artifacts')
         logger.info('Using artifact bucket: {}'.format(s3_bucket))
         self._s3_bucket = s3_bucket
 
-        s3_dir_path = os.environ.get('S3_DIR_PATH', 'autodelete7d')
-        dir_name = '{}-{}'.format(
-            time.strftime("%Y%m%d-%H%M%S"),
-            ''.join([
-                random.SystemRandom().choice(
-                    string.ascii_letters + string.digits) for _ in range(16)
-            ])
-        )
-
-        # Sample s3_directory:
-        # infinity-artifacts/autodelete7d/kafka/20160815-134747-S6vxd0gRQBw43NNy
-        s3_directory_url = os.environ.get(
-            'S3_URL',
-            's3://{}/{}/{}/{}'.format(
-                s3_bucket,
-                s3_dir_path,
-                package_name,
-                dir_name
+        s3_dir_path = os.environ.get('S3_DIR_PATH')
```

Comment:
Hm. So this would allow us to drop the second job entire then, right? We could just have the "build permanent" option in the first job.

Reply:
Yep, I created a `PUBLISH_DCOS` drop down that has three options:

- DISABLED
- AUTODELETE7D
- PERMANENT

It is `DISABLED` by default and yes, no changes to second job.